modules/margaritas.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+function margaritas {
+  checkargn $# 1
+  num=$1
+  re='^[0-9]+$'
+  if ! [[ $num =~ $re ]] ; then
+    echo "Please enter a valid number"
+    exit 1
+  fi
+  if [ "$num" = "" ]; then
+    num=1
+  fi
+  if [ $num -lt 0 ]; then
+    echo "Please enter a valid number of margaritas to print."
+    exit 1
+  fi
+  echo "Here are $num margaritas just for you... :)"
```

Comment:
Could we maybe have an `if` here so that if it's only 1 margarita it could instead print `Here is 1 margarita for you`

Reply:
done